Packages/com.unity.inputsystem/InputSystem/Actions/IInputActionCollection.cs

```diff
@@ -56,11 +57,78 @@ public interface IInputActionCollection : IEnumerable<InputAction>
         /// <summary>
         /// Enable all actions in the collection.
         /// </summary>
+        /// <seealso cref="InputAction.Enable"/>
+        /// <seealso cref="InputAction.enabled"/>
         void Enable();
 
         /// <summary>
         /// Disable all actions in the collection.
         /// </summary>
+        /// <seealso cref="InputAction.Disable"/>
+        /// <seealso cref="InputAction.enabled"/>
         void Disable();
     }
+
+    /// <summary>
+    /// An extended version of <see cref="IInputActionCollection"/>.
+    /// </summary>
+    /// <remarks>
+    /// This interface will be merged into <see cref="IInputActionCollection"/> in a future (major) version.
+    /// </remarks>
+    public interface IInputActionCollection2 : IInputActionCollection
```

Comment:
Perhaps **IInputActionCollectionExtended** would be a better name?
Could some of this be done through extensions so that you can already support IInputActionCollection?

Reply:
`IInputActionCollectionExtended` IMO makes it sound like it's rightfully a different thing from `IInputActionCollection`. Tagging a number on it seemed like the clearest way to mark it as an artifact of versioning.

Reply:
Ah yup, did consider extension methods but unfortunately, at least for some of the stuff here, wasn't an option (e.g. `bindings` can't be implemented properly with just the stuff in `IInputActionCollection`).